### PR TITLE
API Build

### PR DESCRIPTION
**Pull request by vyasworks**

### CODE REVIEW

**Comment by vyasworks** [comment]
Don't know why yarn locks are having large diffs the last few PRs.

File: Makefile [hunk 1]
```diff
@@ -86,3 +89,18 @@ docker-run:
...
+pre-build:
+	@rm -rf .build || true
+	@mkdir -p .build;
+	@rm -rf terraform/.artifacts || true
+	@mkdir -p terraform/.artifacts
+
+api-build: pre-build
+	@yarn install
+	@yarn workspace @symchk/api build
+	@yarn workspaces focus @symchk/api --production
```
Comment (hitankar-freshworks) [inline Makefile:102]: Does this clean up the contents of `node_modules` folder from the yarn install two lines above?

File: Makefile [hunk 2]
```diff
@@ -86,3 +89,18 @@ docker-run:
...
+	@cp -r node_modules .build/node_modules
+	@cp -r ./apps/api/dist .build
```
Comment (hitankar-freshworks) [inline Makefile:104]: Maybe better to copy the contents of the dist folder than the dist folder itself?

File: terraform/api.tf [hunk 3]
```diff
@@ -5,7 +5,7 @@ resource "aws_lambda_function" "api" {
   runtime          = "nodejs14.x"
   filename         = local.api_build_zip
   source_code_hash = filebase64sha256(local.api_build_zip)
-  handler          = "lambda.handler"
+  handler          = "dist/lambda.handler"
```
Comment (hitankar-freshworks) [inline terraform/api.tf:8]: please check my previous comment

File: Makefile (hunk 2 above)
Comment (hitankar-freshworks) [reply Makefile:104]:
```suggestion
	@cp -r ./apps/api/dist/* .build
```

File: Makefile (hunk 1 above)
Comment (vyasworks) [reply Makefile:102]: Yes it will. We need the install for the build to work

**Review by hitankar-freshworks — APPROVED** [review]

**Pull request merged**